test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -415,8 +521,14 @@ void static inline calc_and_print_results(double total_t, int len,
     PE_set_used_adjustments(&nPEs, &stride, &start_pe, metric_info);
 
     if (total_t > 0 ) {
+
+#ifdef ENABLE_OPENMP
+        bw = (len / 1e6 * metric_info.window_size * metric_info.trials *
+                (double)metric_info.nthreads) / (total_t / 1e6);
```

Comment:
I forget C type promotion rules, but I wonder if there could be loss of precision here since this will be evaluated left-to-right (i.e. `len/1e6` generating an integer result).  I would tend to err on the safe side and change those `1e6` literals to `1.0e6` or throw in an explicit type cast on the left-hand side of the expression.

Reply:
(same comment in the `#else` branch of this code)

Reply:
@jdinan Thanks, good suggestion. Will update it.